articles/ros2_access_control_policies.md

```diff
@@ -99,11 +99,32 @@ Attributes:
 - **version**: declared version of schema version in use
   - Allows for advancing future revisions of the schema
 
-### `<profiles>` Tag
+### `<enclaves>` Tag
 
-Encapsulates a sequence of unique profiles.
+Encapsulates a sequence of unique enclaves.
 This method of nesting sequences allows for additional tags to be extended to the `<policy>` root.
 
+### `<enclave>` Tag
+
+Encapsulates a collection of profiles.
+This is specific to a enclave as determined by associative attributes.
+
+Attributes:
+- **path**: Fully qualified enclave path
+
+Given that multiple nodes can be composed into a single process, a enclave is used to contain the collection of profiles of all respective nodes.
+A enclave may therefore be considered the union of contained profiles.
+Note that the union of profiles within a enclave will result in denied privileges of any profile to supersede all allowed privileges for every profile.
```

Comment:
This is difficult to read. What are we trying to say here ? that if a profile asks for a permission but that same permission  has been explicitly denied/blacklisted by another profile of the enclave it will be rejected?

Reply:
Done: d29e06a